ui/components/ui/list-item/list-item.component.js

```diff
@@ -19,7 +19,6 @@ export default function ListItem({
     className,
     subtitle || children ? '' : 'list-item--single-content-row',
   );
-
```

Comment:
nit: any reason for this?

Reply:
I didn't like it there 😄 no not really haha